server/registry/names/spec_revision.go

```diff
@@ -22,8 +22,8 @@ import (
 var specRevisionRegexp = regexp.MustCompile(fmt.Sprintf("^projects/%s/locations/%s/apis/%s/versions/%s/specs/%s(?:@%s)?$",
 	identifier, Location, identifier, identifier, identifier, revisionTag))
 
-var specRevisionCollectionRegexp = regexp.MustCompile(fmt.Sprintf("^projects/%s/locations/%s/apis/%s/versions/%s/specs@$",
-	identifier, Location, identifier, identifier))
+var specRevisionCollectionRegexp = regexp.MustCompile(fmt.Sprintf("^projects/%s/locations/%s/apis/%s/versions/%s/specs/%s@$",
```

Comment:
@theganyo maybe another bug in the regex? I found these writing tests and checking code coverage -- the associated branches weren't being taken until I updated these regexes.

Reply:
Should these be addressed in a separate PR?

Reply:
Maybe, certainly if we had more complete tests on the `names` package #911

Reply:
merged in #912